packages/adapters/github-config/index.json

```diff
@@ -0,0 +1,64 @@
+{
+  "themes": {
+    "LIGHT": "boolean(//html[@data-color-mode='light'])",
+    "DARK": "boolean(//html[@data-color-mode='dark'])"
+  },
+  "contexts": {
+    "GLOBAL": {
+      "containerSelector": "html",
+      "contextBuilder": {
+        "id": "string('global')",
+        "websiteName": "string('GitHub')",
+        "username": "normalize-space(string(//div[@class='Overlay-titleWrap']//span[@class='Truncate-text']))",
+        "fullname": "normalize-space(string(//div[@class='Overlay-titleWrap']//span[@class='Truncate-text']/../../span[2]))",
+        "img": "string(//div[@class='Overlay-titleWrap']//img/@src)",
+        "url": "string(//html/head/meta[@property='og:url']/@content)"
+      }
+    },
+    "PROFILE": {
+      "containerSelector": ".js-profile-editable-replace",
+      "contextSelector": "",
+      "widgets": {
+        "button": {
+          "styles": "styles/profile/button.css",
+          "insertionPoint": "div.clearfix",
+          "insert": "end"
+        },
+        "avatarBadge": {
+          "styles": "styles/profile/avatarBadge.css",
+          "insertionPoint": "div.clearfix > div",
+          "insert": "inside"
+        }
+      },
+      "contextBuilder": {
+        "id": "normalize-space(substring-before(substring-after(string(.//h1/span[2]), '\n'), '\n'))",
+        "authorFullname": "normalize-space(string(.//h1/span[1]))",
+        "authorUsername": "normalize-space(substring-before(substring-after(string(.//h1/span[2]), '\n'), '\n'))",
+        "authorImg": "string(.//img/@src)",
+        "url": "string(//html/head/meta[@property='og:url']/@content)"
+      }
+    },
+    "POST": {
+      "containerSelector": ".js-discussion",
+      "contextSelector": ".TimelineItem.js-comment-container",
+      "widgets": {
+        "button": {
+          "styles": "styles/post/button.css",
+          "insertionPoint": ".timeline-comment-actions",
+          "button": "end"
+        },
+        "avatarBadge": {
+          "styles": "styles/post/avatarBadge.css",
+          "insertionPoint": "div.TimelineItem-avatar > a",
+          "button": "end"
+        }
+      },
+      "contextBuilder": {
+        "id": "string(.//div[contains(@class,'timeline-comment-group')]/@id)",
+        "authorUsername": "substring-after(string(.//div[contains(@class,'TimelineItem-avatar')]/a/@href | .//a[contains(@class,'TimelineItem-avatar')]/@href), '/')",
+        "authorImg": "string(.//*[contains(@class,'TimelineItem-avatar')]//img/@src)",
+        "url": "string(//html/head/meta[@property='og:url']/@content)"
```

Comment:
Should be concatenate of:
* `string(//html/head/meta[@property='og:url']/@content)`
* `#issuecomment-`
* comment id